client/src/components/Hero.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import "./Hero.css";
+import heroImage from "../images/heroImage.png";
+
+const content = {
+	text1: (
+		<div className="subheader">
+			<h1>Black Atlantic Space-Time</h1>
```

Comment:
I'd prefer clearer separation between code and content. Something like:
```js
const content = {
 h1: "Black Atlantic Space-Time",
 }
 
 return <div>
     <h1>{content.h1}</h1>
   </div>
 ```
 
The current version still mixes text and JSX, which I'm not a fan of.